datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/jmx.yaml

```diff
@@ -10,6 +10,15 @@
   value:
     type: integer
 
+- name: is_jmx
+  description: |
+   Whether or not this instance is a configuration for a JMX integration.
+   If `is_jmx` is set to true at the init_config level, this flag is ignored.
+  hidden: true
+  value:
+    example: true
```

Comment:
Default should not be true. For a pure jmx integration it makes sense to have the init config one being true. For a mixed integration I think sample config should be more like:

````
init_config:
instances:
    - is_jmx: true
    - other things that do not even mention jmx
```

Reply:
so you mean having `is_jmx: false` is the `jmx.yaml` spec template, and override its value to true in jmx-only integration?